paper/paper.md

```diff
@@ -145,21 +145,15 @@ _--documentation_: Instead of traversing the whole file, this feature annotates
 
-- Schema Package: The NeXus (meta)data definitions are expressed in XML using the NeXus Definition Language (NXDL), which in turn is defined using XSD. pynxtools converts this representation and extends NOMAD's internal data schema (called __Metainfo__) with these domain-specific quantities. pynxtools also connects the NeXus vocabulary to existing base sections in NOMAD — reusable, standardized building blocks used to represent common scientific concepts. This connection enables interoperability between NeXus-defined concepts and other standardized representations in NOMAD, such as those for sample synthesis or theoretical calculations.
+The __DataConverter__ as described above is available in NOMAD, allowing users to convert their experimental data to NeXus through NOMAD's graphical interface. The __DataConverter__ class also handles NOMAD's electronic lab notebooks (ELNs), converting manual input data to NeXus.
 
-- Data Converter: The __DataConverter__ as described above is also available in NOMAD. Thus, NOMAD users can directly convert their experimental data to NeXus using NOMAD's graphical interface. In addition to the capabilities already described, the internal __DataConverter__ class also handles NOMAD's electronic lab notebooks (ELNs) and converts these such that the manually inputted data can be converted to NeXus as well.
+The NeXus (meta)data definitions are expressed in XML using the NeXus Definition Language (NXDL), which is defined using XML Schema Definition (XSD). pynxtools converts this representation and extends NOMAD's internal data schema (called __Metainfo__) with these NeXus-specific quantities. It also connects the NeXus vocabulary to existing base sections in NOMAD — reusable, standardized building blocks for representing common scientific concepts. This connection enables interoperability between NeXus-defined concepts and other standardized representations in NOMAD, such as those for sample synthesis or theoretical calculations. The NOMAD parser module in pynxtools (__NexusParser__) extracts structured data from NeXus HDF5 files to populate NOMAD with __Metainfo__ object instances as defined by the pynxtools schema package. This enables direct ingestion of NeXus data into NOMAD. Parsed data is post-processed using NOMAD's normalization pipeline. This includes automatic handling of units, linking references (including sample and instrument identifiers defined elsewhere in NOMAD), and populating derived quantities needed for advanced search and visualization. The entire pynxtools workflow (conversion, parsing, and normalization) is exemplified in a representative NOMAD upload shipped with the package. This example helps new users understand the workflow and serves as a template to adapt the plugin to new NeXus applications.
```

Comment:
Why at all to mention that NeXus is XML and XSD, important is there is a __DataConverter__ functionality that can be used command-line or in NOMAD. In the case of "in NOMAD" the __DataConverter__ is called internally plus other functionalities, like reported here.

Everything after "This connection enables" is fine.

I somewhat feel that this connection is also where to mention that ELN data are processed as well into Metainfo.